Gemfile

```diff
@@ -5,37 +5,23 @@ git_source(:github) { |_repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.5'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.1'
-# Use sqlite3 as the database for Active Record
-# gem 'sqlite3', '~> 1.4'
+# gem 'bcrypt', '~> 3.1.7'
+# gem 'image_processing', '~> 1.2'
+# gem 'redis', '~> 4.0'
+gem 'bootsnap', '>= 1.4.2', require: false
+gem 'jbuilder', '~> 2.7'
 gem 'pg'
-# Use Puma as the app server
 gem 'puma', '~> 4.1'
-# Use SCSS for stylesheets
+gem 'rails', '~> 6.0.1'
```

Comment:
I love the impulse towards 🔤. I don't have a thing to cite for this, but it's ok to leave Rails out of that. This is a Rails app and Rails is  dependency we can't really swap out without a significant reorganization/rewrite. I'd think of it as its own group.